driver_cpl/driver/prep_ocn_mod.F90

```diff
@@ -650,6 +653,7 @@ subroutine prep_ocn_merge( flux_epbalfact, a2x_o, i2x_o, r2x_o, w2x_o, g2x_o, xa
        index_x2o_Faxa_snow_16O  = mct_aVect_indexRA(x2o_o,'Faxa_snow_16O' , perrWith='quiet')
        index_x2o_Faxa_prec_16O  = mct_aVect_indexRA(x2o_o,'Faxa_prec_16O' , perrWith='quiet')
        index_x2o_Foxx_rofl_16O  = mct_aVect_indexRA(x2o_o,'Foxx_rofl_16O' , perrWith='quiet')
+       lwiso_runoff = ( index_x2o_Foxx_rofl_16O /= 0 )
```

Comment:
It seems like the way you've done this works, but I'd prefer to see the setting of lwiso_runoff done in prep_ocn_init rather than prep_ocn_merge: Then I'd be more confident that this is definitely set before it is referenced.

Then, optionally, you could use the already-set lwiso_runoff in prep_ocn_merge to determine whether to try to get the various 16O and 18O fields.